### PR TITLE
Hotels: use real photos

### PR DESCRIPTION
**Pull request by mrtnzlml**
Note that SingleHotel component will be completely rewrited so we
can freely ignore it now...

### CODE REVIEW

**Comment by mrtnzlml** [comment]
Please test these changes in [Expo](https://docs.expo.io/versions/latest/introduction/installation.html#mobile-client-expo-for-ios-and-android) after build has finished:

  QR Code

File: app/hotels/src/SingleHotel.js [hunk 1]
```diff
@@ -1,55 +1,91 @@
...
+    <SingleHotel
+      data={propsFromRenderer}
+      onGoToHotelGallery={this.props.onGoToHotelGallery}
+    />
+  );
+
+  render = () => (
+    // TODO: replace with real ID of the hotel
+    <PublicApiRenderer
+      query={graphql`
```
[the remark this answers is not in the document]
Comment (mrtnzlml) [reply app/hotels/src/SingleHotel.js]: This file will be completely replaced anyway... :)

**Comment by mrtnzlml** [comment]
> source={{ uri: data.mainPhoto && data.mainPhoto.thumbnailUrl }} will throw error if thumbnailUrl is null, which is not excluded by GraphQL schema thumbnailUrl: String

You mean because you cannot use `null` in the `Image` component? Well, I should fix it then.. :)

**Comment by jaroslav-kubicek** [comment]
Exactly, `uri` can't be null, although undefined is fine 🤔

**Comment by mrtnzlml** [comment]
https://github.com/kiwicom/react-native-app/pull/63

**Pull request merged**